yaml/roads.yaml

```diff
@@ -32,6 +31,14 @@ global:
       sport: {col: sport}
       man_made: {col: man_made}
       walking_network: {expr: "mz_hiking_network(osm_id)"}
+      bicycle_network: {expr: "mz_cycling_network(tags, osm_id)"}
+      oneway: {col: oneway}
```

Comment:
Yes, we should normalize & take the line direction into account the same as with `oneway`.

Looks like basic set of attributes as `oneway`: 

* https://taginfo.openstreetmap.org/keys/oneway%3Abicycle#values.